.github/CONTRIBUTING.md

```diff
@@ -24,6 +24,12 @@ npm run style -- --fix
 
 ## Running tests
 
+The tests require the files to be compiled, you must first run the babel transpilation :
```

Comment:
```suggestion
The tests require the files to be compiled, you must first run the babel transpilation:
```
May want to remove the space before the colon here.

Reply:
Sorry I'm french 😉